src/doc/trpl/tasks.md

```diff
@@ -140,8 +147,10 @@ single `Receiver` value.  What if our example needed to compute multiple
 results across a number of threads? The following program is ill-typed:
 
 ```{rust,ignore}
-# fn some_expensive_computation() -> int { 42 }
-let (tx, rx) = channel();
+# use std::sync::mpsc;
+# fn some_expensive_computation() -> u32 { 42 }
+
```

Comment:
I suspect this line will appear in the output; might be better to just remove it?